python/src/worker/tasks/trajectory_analysis.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+import backoff
+import requests
+from aws_xray_sdk.core import xray_recorder
+
+from ..config import config
+from ..result import Result
+from ..tasks import Task
+
+
+class getPseudoTime(Task):
```

Comment:
create a different file for each task

Reply:
> create a different file for each task

why different files? Doesn't make sense to have both tasks in the same file, since they're two steps of the trajectory analysis?

Reply:
in many languages is a common practice to have only one class in each file, and to name the file with the class name. This makes it very easy to open files by name and see how many different tasks are there (just looking at the number of files and names).  Moreover, short files are easier to debug and test, specially in this case where the classes are independent (even if they are part of the same functionality)